src/VisualStudio/VisualBasic/Impl/LanguageService/VisualBasicPackage.vb

```diff
@@ -34,6 +34,10 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic
     <ProvideLanguageExtension(GetType(VisualBasicLanguageService), ".frm")>
     <ProvideLanguageExtension(GetType(VisualBasicLanguageService), ".pag")>
     <ProvideLanguageExtension(GetType(VisualBasicLanguageService), ".vb")>
+    <ProvideLanguageCodeExpansion(GetType(VisualBasicLanguageService), "Basic", 101, "VB", "%InstallRoot%\VB\Snippets\%LCID%\SnippetIndex.xml",
```

Comment:
101 is already defined as "Visual Basic" which is the expected name. It's weird for C# because the expected name is "Visual C#" which apparently isn't a name that's used anywhere else in the package resources.